plugins/woocommerce/client/blocks/assets/js/base/components/skeleton/patterns/product-notice/index.tsx

```diff
@@ -0,0 +1,14 @@
+/**
+ * Internal dependencies
+ */
+import { Skeleton } from '../../';
+
+export const ProductNoticeSkeleton = () => {
```

Comment:
I confused this with actual notices, but this appears to be for form options. Why was this name chosen I wonder?

Reply:
I see where the confusion is coming from. Initially, we were showing a notice, where the current skeleton appears. In  #8842, we changed the notice with a skeleton. That said, the name is indeed misleading. As this part shows the product short description, I changed the name to `ProductShortDescriptionSkeleton` in https://github.com/woocommerce/woocommerce/pull/56719/commits/674ded138f5c3756fabd33262d8fbae96c31c4e9.